fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/JobsServlet.java

```diff
@@ -463,9 +463,8 @@ protected void handleResultsResponse(
                 .subscribe(
                         new HttpResponseChannel(
                                 asyncResponse,
-                                httpResponseMaker,
-                                apiRequest.getFormat(),
-                                apiRequest.getUriInfo()
+                                apiRequest,
```

Comment:
[We also need to inject the `HttpResponseMaker` into the `JobsServlet`](https://github.com/yahoo/fili/pull/447/files#diff-33f581382b3b5c860a25432e57a66854R408)